### ISSUE TITLE
[Windows Terminal>About]: 'About' text should be defined as a heading.

### ISSUE THREAD

**Issue by Saurabh220304** [microsoft/terminal#19686]
### Windows Terminal version

1.25.3571.0

### Windows build number

Windows 11 Version 26H1 (OS Build 28000.1340)

### Steps to reproduce

**Repro Steps:**
1.Launch the Window terminal app.
2.Navigate to the 'New Open Tab' and select the 'About' menu item.
3.About pane will open.
4.Now turn on the narrator using 'CTRL+Win+Enter' keys.
5.Now navigate in scan mode and pressing 'H' key to check the headings on the pane.
6.Observe the issue here.

**User experience:**
Users who rely on screen reader will feel difficulty in analyzing the structure of the page, if visually appearing heading is not defined as heading programmatically.

**WCAG Reference link:**
[1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)

**Attachements:**
https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-heparihar_microsoft_com/IQB-F27Xc9vzTJ8pEBOYbmnaAcaa7btDhF3V6UwPiLrnzxI?e=fLHKRQ

### Expected Behavior

The text “About” should be defined in proper heading level (H), so that assistive technology users can navigate to it easily.

### Actual Behavior

'About' text is styled to look like a heading (e.g., larger font, bold) but is not marked up as a heading. Screen readers do not identify it as a heading, making it inaccessible for users who rely on heading navigation.

**Comment by carlos-zamora** [comment]
Issue repros on the ContentDialog in the WinUI 2 Gallery. Filed an issue upstream on WinUI: https://github.com/microsoft/microsoft-ui-xaml/issues/10946

Considering this a /duplicate of https://github.com/microsoft/microsoft-ui-xaml/issues/10946
